src/ccall.cpp

```diff
@@ -1975,7 +1977,11 @@ jl_cgval_t function_sig_t::emit_a_ccall(
             } else {
                 // since we aren't saving this code, there's no sense in
                 // putting anything complicated here: just JIT the function address
-                llvmf = literal_static_pointer_val(symaddr, funcptype);
+                // get the string name and pointer for the ccall instruction
+                ccallName = MDString::get(ctx.builder.getContext(), symarg.f_name);
+                Constant * temp = literal_static_pointer_val(symaddr, funcptype);
+                ccallAddr =  ConstantAsMetadata::get(cast<ConstantInt>(temp->getOperand(0)));
```

Comment:
Why do we need the `ccallAddr`?

Reply:
we don't, removed and subsequent unnecessary stuff changed